### PR TITLE
sys-kernel/ck-sources: bumps @ LTS (4.9.x / 4.14.x), stable (4.18.x) branches

### PR DESCRIPTION
**Pull request by kuzetsa**
latest genpatches version for respective branches

QA boot tested using "make defconfig" and qemu:

---

```
[    1.085650] MuQSS CPU scheduler v0.150 by Con Kolivas.
[Thu, 20 Sep 2018 07:40:33 +0000] boot complete.
Linux (none) 4.9.128-ck #1 SMP Thu Sep 20 03:16:58 EDT 2018
x86_64 x86_64 x86_64 GNU/Linux
[Thu, 20 Sep 2018 07:40:33 +0000] powering off.
[    1.675468] reboot: Power down
```

---

```
[    1.081569] MuQSS CPU scheduler v0.162 by Con Kolivas.
[Thu, 20 Sep 2018 07:40:36 +0000] boot complete.
Linux (none) 4.14.71-ck #1 SMP Thu Sep 20 03:07:55 EDT 2018
x86_64 x86_64 x86_64 GNU/Linux
[Thu, 20 Sep 2018 07:40:36 +0000] powering off.
[    1.671342] reboot: Power down
```

---

```
[    0.964509] MuQSS CPU scheduler v0.173 by Con Kolivas.
[Thu, 20 Sep 2018 07:40:38 +0000] boot complete.
Linux (none) 4.18.9-ck #1 SMP Thu Sep 20 02:02:18 EDT 2018
x86_64 x86_64 x86_64 GNU/Linux
[Thu, 20 Sep 2018 07:40:38 +0000] powering off.
[    1.571991] reboot: Power down
```

### CODE REVIEW

**Comment by gktrk** [comment]
@kuzetsa can you insert `Signed-off-by:` to commit messages please?

**Comment by kuzetsa** [comment]
> @kuzetsa can you insert `Signed-off-by:` to commit messages please?

@gktrk For two reasons, I don't believe this would be proper:

### quote ###

*Committers must use their real name, i.e., the name that  
would appear in an official document like a passport.* (from GLEP 76)

I'm not inclined to follow this "real name" policy.

... Fortunately, I'm not a committer:

---

A distinction between authors and committers was
made very publicly on the gentoo-dev mailing list:

https://archives.gentoo.org/gentoo-dev/message/fe2a3e900c33bf39747f774f32cd482c

GLEP 76 specifically mentions committers
(*and the author field is not the commit field*)

```
git log --author="kuzetsa" | wc -l
1640

git log --committer="kuzetsa" | wc -l
0 (zero)
```

**Comment by gktrk** [comment]
@kuzetsa thanks for the clarification. I understand that the new copyright policy is frustrating. I know that you switched to using "kuzetsa" a long time ago and we should respect that. Let me know when we're ready to merge and I'll take care of the rest.

**Comment by Zlogene** [comment]
As far as I know it is not really legal as we stopped making exceptions by now. It is also obivious that you do not really have to provide docs justification (later? Who knows), but not using obivious pseudonym as well.

**Comment by kuzetsa** [comment]
> As far as I know it is not really legal as we stopped making exceptions by now. It is also obivious that you do not really have to provide docs justification (later? Who knows), but not using obivious pseudonym as well.

I had difficulty understanding this, @Zlogene.

"it" - what are you commenting on?

"it" is "not really legal"?

**Comment by mgorny** [comment]
I've passed this over to our 'legal' people. Please don't merge until they answer.

File: sys-kernel/ck-sources/metadata.xml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
   <maintainer type="person">
-    <email>kuzetsa@gmail.com</email>
```
[the remark this answers is not in the document]
Comment (kuzetsa) [reply sys-kernel/ck-sources/metadata.xml]: [ [example / citation](https://www.americanbar.org/publications/gpsolo_ereport/2014/october_2014/how_to_ditch_that_unprofessional_email_address.html) ] writer at americanbar.org who starts with:

["... Lawyers, you really do need a professional email address ..."],
following up with a well-reasoned explanation for the valid concerns it raises when a professional isn't aware of, or is willing to remedy the bad impression it gives.

I'm not inclined to say who (*or how many people*) listed in the gentoo.org LDAP had similar (*valid*) criticisms.

Optics are important for a brand, and criticism of gmail's mobile and web clients seems odd when you consider their reputation for reliability. ::shrug::

**Comment by mgorny** [comment]
@kuzetsa, your request has been discussed in great detail throughout the day by the copyright working group, and they have decided to deny it for the reasons outlined below. I'm sorry but we can't accept GCO sign-off via a 'service mark' or any other name that is not a natural person's real name.

I'm sorry that the wording of GLEP 76 have been imprecise, and might have suggested that any registered name confirmed by an official document would suffice. However, the intent of the policy was to use **natural person's real name**, and not trademark or company names. The author of the wording presumed that the mention of passport would suffice to clarify it but apparently it didn't. In order to avoid such mistakes in the future, an update to the wording has been prepared and submitted for public review.

I should point out that one of the points of the policy is that it applies equally to everyone, and therefore we grant no exceptions, neither implicit nor implied by specific local legislation. To provide a better reference: a name can be used for sign-off only if a. your government issued (or would issue) a passport with that specific name, and b. that password would be equally recognized by other countries as passport with your birth name.

I'm sorry but if you can't fulfill this legal requirement, the @gentoo/proxy-maint team can not proxy the commits for you.

**Comment by kuzetsa** [comment]
> @kuzetsa, your request has been discussed [...] or any other name that is not a natural person's real name.
> 
> I'm sorry that the wording of GLEP 76 have been imprecise, and [...] **natural person's real name**, and not trademark or company names. [...] an update to the wording has been prepared and submitted for public review.
> 
> I should point out that one of the points of the policy is that it applies equally to everyone, and therefore we grant no exceptions, neither implicit nor implied by specific local legislation. To provide a better reference: a name can be used for sign-off only if a. your government issued (or would issue) a passport with that specific name, and b. that passport would be equally recognized by other countries as passport with your birth name.
> 
> I'm sorry but if you can't fulfill this legal requirement, the @gentoo/proxy-maint team can not proxy the commits for you.

Thanks for having a look / legal clarification for how personhood is defined.
I'd like to make a suggestion for future communication though though:

["... *equally recognized by other countries as passport with your birth name.*"]

birth name misunderstands legal name issues in a completely different way:

in many jurisdictions, the department of state for a particular nation (*such as the united states, where the gentoo foundation itself is legally resident*) allows for people to affirm a chosen name for passport identification, as well as other federal purposes. This practice applies, for example, to LGBTQ persons who have filed paperwork for legal recognition, as well a customary practice in traditional marriages - recognition when a married person adopts the surname of their spouse (*a woman who marries Mr. Jones might use the name Mrs. Jones, and it is a legally recognized practice to do so without any expectation to request a court order or seek an amendment to her birth record, etc.*)

Reviewed today's update (*-dev ML*), and the language is sensible:

amended language from GLEP 76: Copyright Policy [v4] seems ok,  
and doesn't contain any form of confusing "birth name" language.  
(*claiming equal impact would be tenuous, as mentioned above*)

---

On a personal note, my "legal name as a natural person" is something I had considered changing, but due to cultural bias about names (*minimum of 2-names, with a given name / christian name in addition to a surname as the cultural expectation, as court orders for mononymous names are nearly always denied*) is the main reason I opted to formally register my trade name as a business entity. Rushing the paperwork over the weekend was specifically because of the GCO / signing expectation, and I had sincerely hoped copyright / trademark & contract-type protections (*ability to seek injunctive relief, or to have a lawsuit filed against kuzetsa in the "business entities are people" sense*) would be sufficient. (*it really wasn't a hasty decision, just hasty timing: I'd been planning this for quite a while*)...

Guess there's no getting around it:

Will I be required to sign as Sarah J. White?  
It's awkward that my legal initials are SJW, so if it's permitted I'd prefer to  
write the GCO signoff name as Sarah White (*no middle name or initial*)

Also, I think "and others" should've been used in copyright line,  
as I've just remembered PYTHON_COMPAT=( python2_7 ) is something  
which I don't maintain or test (*specifically used by the deblob feature*)

GLEP 76 doesn't mention metadata.xml - any concern other than GCO?

**Comment by mgorny** [comment]
I just meant that if a country would recognize a passport with your birth name, it should also recognize the one you'll get with that particular name.

As far as I'm aware, middle name isn't really required to be used in western culture. I know people these days sometimes decide to use it (probably because two names aren't sufficient for unique identification anymore) but it's their choice. So we're entirely fine with two names.

Please update the commits and we'll merge them.

**Comment by kuzetsa** [comment]
> Please update the commits and we'll merge them.

in progress.

mangling some keys to reflect legal name which is ironically, more anonymous.

**Comment by gktrk** [comment]
Is the use of name `kuzetsa` in the copyright line of ebuilds intentional or did you simply forget to update them?

**Comment by kuzetsa** [comment]
> Is the use of name `kuzetsa` in the copyright line of ebuilds intentional or did you simply forget to update them?

> Also, I think "and others" should've been used in copyright line,
> as I've just remembered PYTHON_COMPAT=( python2_7 ) is something
> which I don't maintain or test (_specifically used by the deblob feature_)

^ got burried in the other stuff.

I did't forget.

it's updated, and it also complies with polisy (as-written)

GLEP 76: Copyright Policy [v4]:

> This is not a change of policy, but merely a clarification of the real name requirement:

> - A copyright holder can be a legal entity (e.g., a company) in some jurisdictions.

---

I don't know of any other changes being requested.

I've reviewed these commits, used GCP sign-off in the requested manner in a way which is compliant with the legal situation in my jurisdiction as well as policy

I'd like to move on please (*ready to merge*)

File: sys-kernel/ck-sources/metadata.xml [hunk 2]
```diff
@@ -2,8 +2,8 @@
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
```
Comment (mgorny) [inline sys-kernel/ck-sources/metadata.xml]: Technically speaking, GLEP 76 doesn't care about `<name/>` in metadata.xml, so you can put whatever you want there. The important point is just the e-mail address update **which should have been merged before you changed it on Bugzilla**.

**Comment by mgorny** [comment]
I'm also fixing the commit messages to include version bumped to.

**Pull request closed without merging**